### PR TITLE
Adds a text-only classroom guide

### PR DESCRIPTION
**Pull request by ryaholm**
Adds a text-only classroom creation guide. (One of the teachers at PD requested it).

🍚 🌭 🍔

### CODE REVIEW

File: rpi_csdt_community/templates/rpi_csdt_community/classrooms_guide.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
...s, to address their needs as facilitators. CSDTs are set up to simulate a cloud computing experience. Each user can save their program to the cloud, which shows up under their profile even if they're not in a classroom. The purpose of classrooms is to organize this cloud computing into a sort of gallery or boutique for all the students in the classroom, including the teacher. With a classroom, students can create and save their programs, then submit them for approval by the teacher or teachers. The teacher approves projects using a visual representation of the program, to verify that it is school appropriate, and then all approved projects are added to the pool of works created by the classroom. This pool of shared works can be viewed by other students, commented on, and even remixed by other students using the other's person's source code. Classrooms allow for sharing between students and at-a-glance observation of who is creating what, providing motivation and teamwork. For the teacher, it permits an easy source to view who is keeping up with the tutorials, who is struggling, who is completing assignments, who is succeeding, and who is creating genuinely interesting works of art.
```
Comment (SillyInventor) [inline rpi_csdt_community/templates/rpi_csdt_community/classrooms_guide.html]: Teachers != students: "boutique for all the students in the classroom" -> "boutique for all the members of the classroom"

Doing more than tutorials "who is keeping up with the tutorials" -> "Who is keeping up with their work"

File: rpi_csdt_community/templates/rpi_csdt_community/classrooms_guide.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
...atements to return a variety of text replies based on the range of an input number, for example. Not to say those types of tutorials aren't important, but CSDTs let your students make colorful, animated, visually engrossing pieces that can be shared with others in your class. The Classrooms app lets your students share what they make, so that everyone can see what range of colors inspires whom and learn collectively to make better and better programmable art. It turns programming into a discussion group, a sort of forum where students can peek inside other project's code to see how things are created, then adapt that code for their own projects. I see classrooms as facilitating a sort of "critique time" where every student hangs their art on the wall, and the class as a whole gets to critique each piece one by one and discuss their emotional reactions to the piece. Just as this helps a developing artist, the same is true for beginning programmers. Our CSDT programs are designed to facilitate focus on the cultural aspects of programming, and to turn your computer lab into an art studio through the Classrooms app goes beyond the normal experiences of introductory programming courses.
```
Comment (SillyInventor) [inline rpi_csdt_community/templates/rpi_csdt_community/classrooms_guide.html]: Delete, they don't need to know why you like CSDTs... this is a guide for class creation.

File: rpi_csdt_community/templates/rpi_csdt_community/classrooms_guide.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            </p>
+            <p>
+                When a student requests to join your class, a request notification will be sent to your administration panel. To go to your administration panel, go to the classrooms page. Your own classrooms will be at the top of the list. Click "Admin" to the right of your classroom title. Under the member-request panel, all people who have asked to join your classroom await approval. To group approve, click their checkmark above their icon for as many as your want to approve, then under the "action" dropdown choose "Approve" and click "Submit all changes". They will promoted to the "students" panel. As a student, when they save their project, there will be a bar where they can choose whether to save to their classroom. They will always want to use this option, which requires them to highlight the name of their classroom. Teach them how to use this feature to prevent projects being saved to the public pool instead of your classroom pool.
```
Comment (SillyInventor) [inline rpi_csdt_community/templates/rpi_csdt_community/classrooms_guide.html]: Awkward phrasing: "To group approve, click their checkmark above their icon for as many as your want to approve," -> "To approve students, click the checkmark above their icon"

File: rpi_csdt_community/templates/rpi_csdt_community/classrooms_guide.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+            </p>
+            <p>
+                After a student publishes a project to their classroom, it will show up under the "Approval-requests" panel in the classroom administrator. It is not visible to the students yet. Here you can see a screenshot of the project, to deem whether it is classroom appropriate or not. If it is, you can check it and then under the "action" dropbox choose "approve". This will promote it to the "approved-objects" pool, where it will be visible to other students in the classroom. It will display their name underneath the project as well as a description, if the student inputs one. This allows you to curate the presentation of projects, so that what is seen by your students is verified appropriate. We at CSDT do not approve projects, so it will be your responsibility to check in to approve new projects if you do not want them to languish in the approval queue.
```
Comment (SillyInventor) [inline rpi_csdt_community/templates/rpi_csdt_community/classrooms_guide.html]: Awkward: "in the classroom administrator" -> "of the classroom administration page"
Don't go by just the photo: "Here you can see a screenshot of the project" -> "Here you can see a screenshot, or click to view the project as a whole."

**Comment by ryaholm** [comment]
Okay, fixed according to requests.

**Pull request merged**